tests/operator_tests.py

```diff
@@ -476,13 +467,79 @@ def test_pda_power(self):
         ex = np.power(n, [2, 3, 4])
         self.assertListEqual(p.to_list(), ex.tolist())
 
+        # Test a singleton with and without a Boolean argument
+        n = np.array([7])
+        a = ak.array(n)
+
+        p_True = ak.power(a, 3, True)
+        p_Imply = ak.power(a, 3)
+        p_False = ak.power(a, 3, False)
+        self.assertListEqual(p_True.to_list(), p_Imply.to_list())
+        self.assertListEqual(p_False.to_list(), a.to_list())
+
+        # Test an with and without a Boolean argument, all the same
+        n = np.array([0, 0.0, 1, 7.0, 10])
+        a = ak.array(n)
+
+        truth_True = ak.array([True, True, True, True, True])
+        truth_False = ak.array([False, False, False, False, False])
```

Comment:
same comment about not needing to declare a var, but I also wanted to highlight the `ak.ones` and `ak.zeros` functionality which i think will simplify this a bit

```python
>>> ak.ones(5, bool)
array([True True True True True])

>>> ak.zeros(5, bool)
array([False False False False False])
```